config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -533,3 +584,47 @@ periodics:
       - --test_args=--ginkgo.focus=\[Feature:SCTP\] --ginkgo.skip=\[Feature:NetworkPolicy\]
       - --timeout=50m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20201203-4778e22-master
+
+- interval: 12h
+  name: ci-kubernetes-e2e-ubuntu-gce-network-policies
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=70
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env=ALLOW_PRIVILEGED=true
+      - --env=NETWORK_POLICY_PROVIDER=calico
+      - --env=KUBE_CONTAINER_RUNTIME=containerd
+      - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.4.0
+      - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.0-rc92
+      - --env=LOG_DUMP_SYSTEMD_SERVICES=containerd
+      - --env=CONTAINER_RUNTIME_TEST_HANDLER=true
+      - --env=KUBE_MASTER_OS_DISTRIBUTION=ubuntu
+      - --env=KUBE_GCE_MASTER_IMAGE=ubuntu-2004-focal-v20200423
+      - --env=KUBE_GCE_MASTER_PROJECT=ubuntu-os-cloud
+      - --env=KUBE_NODE_OS_DISTRIBUTION=ubuntu
+      - --env=KUBE_GCE_NODE_IMAGE=ubuntu-2004-focal-v20200423
+      - --env=KUBE_GCE_NODE_PROJECT=ubuntu-os-cloud
+      - --ginkgo-parallel=30
+      - --gcp-master-image=ubuntu
+      - --gcp-node-image=ubuntu
+      - --gcp-nodes=4
+      - --gcp-zone=us-west1-b
+      - --provider=gce
+      - --test_args=--ginkgo.focus=\[sig-network\]|\[Conformance\] --ginkgo.skip=\[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS)\]|DualStack
```

Comment:
wait, dont we need `NetworkPolicy` in here?

Reply:
is not included under the sig-network tag?

Reply:
it is
 ```
[sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between server and client 
  should allow ingress access from updated namespace [Feature:NetworkPolicy]
•{"msg":"PASSED [sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between server and client should allow ingress access from updated namespace [Feature:NetworkPolicy]","total":188,"completed":159,"skipped":4718,"failed":0}
[sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between serve
```

Reply:
```➜  kubernetes git:(kubeproxy-config-todo) ✗ KUBECONFIG=/Users/jayunit100/.kube/config .//_output/local/go/bin/e2e.test --ginkgo.focus="sig-network|Conformance" --ginkgo.skip="Networking-IPv6|Example|Federation|PerformanceDNS|DualStack" --dump-logs-on-failure=false --ginkgo.dryRun=true | grep Ran
```
Results in 
```
Ran 491 of 5700 Specs in 0.035 seconds
```
Is that really what you want to do ?  Thats a huge subset of tests.  I feel like the goal was to get a **fast** running, clear networkpolicy job, not a job with every single sig-network test

Reply:
> ( i do see your parallelizing @ 30 but... still ... the signal will be way bigger then what we wanted for a networkpolicy conformance workflow )

the biggest risks with network policies ,and the most difficult bugs I've seen so far, are because they affect other traffic in the cluster ... the network policies must not affect the other tests and it will be a big problem if they affect a conformance test...

> . I feel like the goal was to get a fast running, clear networkpolicy job, not a job with every single sig-network test

the job pull-kubernetes-e2e-gce-ubuntu-containerd  runs more than 800 tests in 40 mins, this should be around 20 mins, anyway, it will be always less than the `pull-kubernetes-e2e-gce-ubuntu-containerd` presubmit job that runs in all PRs, there is no a big benefit in being superfast AFAIK

Reply:
ahhh ok  .  id still put `NetworkPolicy` in the ginkgo focus just to clarify that this is the purpose of the job, but that is just a nit.  otherwise i guess this will work for the goal your suggesting. 

ok makes sense carry on :)